FakeStorePathに~/を指定するとpanicする

以下のようなプロファイルを与えるとpanicする。

```js
{
    "AccessToken": "<token>",
    "AccessTokenSecret": "<secret>",
    "FakeMode": true,
    "FakeStorePath": "~/.usacloud/fake_store.json"
}
```

エラーメッセージ:
```console
panic: open ~/.usacloud/fake_store.json: no such file or directory
```

--- a/go.mod
+++ b/go.mod
@@ -15,7 +15,7 @@ require (
 	github.com/mattn/go-runewidth v0.0.6 // indirect
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/olekukonko/tablewriter v0.0.0-20180506121414-d4647c9c7a84
-	github.com/sacloud/libsacloud/v2 v2.9.1
+	github.com/sacloud/libsacloud/v2 v2.9.3
 	github.com/skratchdot/open-golang v0.0.0-20200116055534-eef842397966
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5

--- a/pkg/config/init_flags_wasm.go
+++ b/pkg/config/init_flags_wasm.go
@@ -1,3 +1,17 @@
+// Copyright 2017-2020 The Usacloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 // +build wasm
 
 package config

